build.sbt

```diff
@@ -38,6 +38,7 @@ lazy val microservice = Project(appName, file("."))
   .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin)
   .disablePlugins(JUnitXmlReportPlugin) //Required to prevent https://github.com/scalatest/scalatest/issues/1427
   .settings(
+    onLoadMessage := "",
```

Comment:
I was expecting to see some additional settings added here for the logging?

Reply:
I tried, but it clashes with the flex dependency for some reason.